packages/commcare/test/index.js

```diff
@@ -97,3 +97,50 @@ describe('SubmitXls', () => {
     expect(formdata.get('create_new_cases')).to.equal('on');
   });
 });
+
+describe('getCases', () => {
+  it('should fetch cases', async () => {
+    const domain = 'my-domain';
+    const app = 'my-app';
+
+    let method;
+    let path;
+
+    testServer
+      .intercept({
+        path: `/a/${domain}/api/v0.5/case`,
+        method: 'GET',
+      })
+      .reply(200, req => {
+        // save out the form data that was upladed
+        method = req.method;
+        path=req.path;
+
+        // simulate a return from commcare
+        return { code: 200, message: 'success' };
```

Comment:
Can we maybe include some sample case data here?

Like if I call `get('/case')`, where does my actual case data end up?

Reply:
Is this even what the cases data looks like? If I check [the docs ](https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143957300/List+Cases+version+3#JSON-Format) the json response looks very different